### PR TITLE
Paper additions and fixes.

### PR DESCRIPTION
**Pull request by nagelt**
Here are some small improvements. I also added some conclusions.

### CODE REVIEW

File: docs/papers/joss/paper.md [hunk 1]
```diff
@@ -182,22 +183,22 @@ studies:
   which use different solvers for independent assessment and review.
   From the same `MFront` source file, the `MFrontGallery` can generate
   shared libraries for all the solvers of interest. Moreover, the
-  project allows publishes a guidline of the [best practices
```
Comment (thelfer) [inline docs/papers/joss/paper.md:185]: publishing ?

File: docs/papers/joss/paper.md [hunk 2]
```diff
@@ -157,7 +158,7 @@ in different forms. Here, the following categorization is employed:
   state of the material.
 - **Behaviours** describe how a material evolves and reacts locally due
   to gradients inside the material. Here, the material reaction is
-  associated with fluxes (or forces) thermodynamically conjugated with
+  associated with fluxes (or forces) thermodynamically conjugate to
```
Comment (thelfer) [inline docs/papers/joss/paper.md:161]: conjugated ?

**Review by thelfer — CHANGES_REQUESTED** [review]
Just a few question

File: docs/papers/joss/paper.md (hunk 1 above)
Comment (nagelt) [reply docs/papers/joss/paper.md:185]: publishes (or 'has')

**Comment by nagelt** [comment]
> Just a few questions. Also, did you reduce the number of references to OpenGeoSys ?

I did  .. it's down to two now (Kolditz et al. 2012 and Bilke et al. 2019)

File: docs/papers/joss/paper.md (hunk 2 above)
Comment (nagelt) [reply docs/papers/joss/paper.md:161]: I like conjugate to, as it's more formal than using the participle ...

**Pull request merged**